src/css/main.scss

```diff
@@ -133,3 +133,43 @@ button {
 .pink-background-bottom-shape {
   bottom: -1%;
 }
+
+//Contact
+.contact-page {
+  padding-left: $site-padding-x-mobile;
+  padding-right: $site-padding-x-mobile;
+
+  @include media('sm-up') {
+    padding-left: $site-padding-x-tablet;
+    padding-right: $site-padding-x-tablet;
+  }
+
+  @include media('md-up') {
+    padding-left: $site-padding-x-desktop;
+  }
+}
```

Comment:
these settings seem pretty generic, if you'll use them again for another page, maybe create a more generic class

Reply:
The `md-up` padding only applies to this page but I ended up using a class I had already set up for the `site-padding-x` and just making it `padding-right: 0` on screens `md-up` :)

Reply:
ah, I see. you can always `@extend` from `site-padding-x` too ;)